core/pom.xml

```diff
@@ -145,13 +145,13 @@
       <scope>test</scope>
     </dependency>
     <dependency>
-      <groupId>org.easymock</groupId>
-      <artifactId>easymock</artifactId>
+      <groupId>org.apache.logging.log4j</groupId>
+      <artifactId>log4j-slf4j-impl</artifactId>
```

Comment:
A log4j package with "impl" in the name makes me think this isn't public API.  Is this for public use?

Reply:
It is not public API. This is the "slf4j-api" implementation that bridges to the "log4j2-api" at runtime. It is needed on the runtime classpath.